raspberry_pi_experiments/wattsup_server.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+import sys, socket
+import os, serial
+import json
+import argparse
+
+from flask import Flask
+from threading import Thread
+
+EXTERNAL_MODE = 'E'
+INTERNAL_MODE = 'I'
+TCPIP_MODE = 'T'
+FULLHANDLING = 2
+
+app = Flask(__name__)
+meter = None
+
+class WattsUp(object):
+    def __init__(self, port, interval=1.0):
+        self.serial = serial.Serial(port, 115200)
```

Comment:
Is the 115200 always fixed?

Reply:
Yes, Im pretty sure it's fixed. I will add a constant for it